Ginger/GingerCore/Repository/RepositoryItem.cs

```diff
@@ -118,57 +118,7 @@ public void InvokPropertyChanngedForAllFields()
             }
         }
 
-        public static string GetShortType(Type T)
-        {
-            //Not so much nice clean OO design but due to static on derived limitation, meanwhile it is working solution 
-            // Put here only class which are saved as stand alone to file system
-            // TODO: add interface to classes which are saved as files which will force them to im
-
-            //TODO: more safe to use type of then Full Name - fix it!
-            if (T == typeof(BusinessFlow)) { return "BusinessFlow"; }
-            if (T == typeof(ActivitiesGroup)) { return "ActivitiesGroup"; }
-            if (T == typeof(Activity)) { return "Activity"; }
-            if (T == typeof(ErrorHandler)) { return "Activity"; }
-            if (T == typeof(CleanUpActivity)) { return "Activity"; }
-            if (typeof(Act).IsAssignableFrom(T)) { return "Action"; }
-            if (typeof(VariableBase).IsAssignableFrom(T)) { return "Variable"; }
-            if (typeof(DataSourceBase).IsAssignableFrom(T)) return "DataSource";
-            if (T.FullName == "GingerCore.Agent") return "Agent";
-            if (T.FullName == "GingerCore.Environments.ProjEnvironment") return "Environment";
-            if (T.FullName == "Ginger.Run.RunSetConfig") return "RunSetConfig";
-            if (T.FullName == "Ginger.Run.BusinessFlowExecutionSummary") return "BusinessFlowExecutionSummary";
-            if (T.FullName == "Ginger.Reports.ReportTemplate") return "ReportTemplate";
-            if (T.FullName == "Ginger.Reports.HTMLReportTemplate") return "HTMLReportTemplate";
-            if (T.FullName == "Ginger.Reports.HTMLReportConfiguration") return "HTMLReportConfiguration";
-
-            if (T.FullName == "Ginger.Reports.HTMLReportConfiguration") return "HTMLReportConfiguration";
-            if (T.FullName == "Ginger.TagsLib.RepositoryItemTag") return "RepsotirotyItemTag";
-
-            if (T == typeof(ApplicationDBTableModel)) return "ApplicationDBTableModel";
-            if (T == typeof(ApplicationDBModel)) return "ApplicationDBModel";
-            if (T == typeof(ApplicationPOMModel)) return "ApplicationPOM";
-
-            // Make sure we must impl or get exception
-            throw new Exception("Unknown Type for Short Type Name " + T.Name);
-        }
-
-        public new string ObjFileExt { get { return FileExt(this.GetType()); } }
-
-        public new static string FolderName(Type T)
-        {
-            string s = GetShortType(T) + "s";
-            if (s.EndsWith("ys"))
```

Comment:
how you handled the "ies" if needed, or the handling of @"SharedRepository\" below?

Reply:
FolderName Method is only used in get of ObjFolderName. And ObjFolderName have 0 references. 

Also we have same field on RepositoryItemBase. It is handled from there.